providers/buffered/provider.go

```diff
@@ -0,0 +1,64 @@
+// © 2017 Helix OpCo LLC. All rights reserved.
```

Comment:
This is a public repo with an MIT license. Should these comments be in the files?
cc: @chriswhelix  @helixjim

Reply:
Good question, didn't realize this repo was public.

Reply:
Well, it's still copyrighted.